src/middleware.ts

```diff
@@ -54,7 +54,9 @@ export function createMiddleware(
             res.setHeader(name, value)
           })
 
-          res.send(body)
+          setTimeout(() => {
+            res.send(body)
+          }, delay || 0)
```

Comment:
Since, technically, `0` timeout puts the code execution to the next even loop (almost synonymous with `nextTick()`), I'd recommend conditionally using `setTimeout` instead so we properly cover the cases when no timeout was intended.

```suggestion
if (delay) {
  setTimeout(() => res.send(body), delay)
  return
}

res.send(body)
```

Reply:
That makes sense.